.circleci/config.yml

```diff
@@ -58,14 +58,18 @@ jobs:
       - run: etc/testing/circle/build.sh 
       - run: etc/testing/circle/launch.sh 
       - run: etc/testing/circle/run_tests.sh 
-      - save_cache:
-         key: pach-go-mod-cache-v2-{{ checksum "go.sum" }}
-         paths:
-         - /home/circleci/.go_workspace/pkg/mod
-      - save_cache:
-         key: pach-go-build-cache-v1-{{ .Branch }}-{{ checksum "current_week" }}-{{ .BuildNum }}
-         paths:
-         - /home/circleci/.gocache
+      - when:
```

Comment:
pick one bucket to save cache from?

Reply:
Not sure I follow - this only does these steps when the bucket is MISC. The syntax is wild.